css/style.css

```diff
@@ -0,0 +1,279 @@
+.feeds {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  padding: 4rem 8rem;
+  background: #ffffff;
+}
+.feeds__right {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.feeds__contents--start {
+  margin-top: 6rem;
+}
+.feeds__linear {
+  border-top: 8px solid #e74040;
+  padding: 1rem 0;
+}
+.feeds--write {
+  background: #ffffff;
+}
+.btn--outlined {
+  padding: 0 !important;
+  display: flex;
+  align-items: center;
+  gap: 1.2rem;
+  margin-top: 1.2rem;
+  font-size: 18px;
+}
+.action__button {
+  margin-top: 1.2rem;
+}
+.primary-color {
+  color: #1ba0ab;
+  font-weight: bold;
+}
+.feeds__contents {
+  align-content: flex-start;
+  padding: 4rem 8rem;
+}
+.feeds__contents--start {
+  margin: auto 0;
+}
+.feeds__title {
+  font-size: 16px;
+}
+.feeds__subtitle {
+  font-size: 3rem;
+  margin: 0;
+  color: #252b42;
+  margin-bottom: 20px;
+}
+
+.teams-section {
+  margin-top: 3rem;
+  background: #ffffff;
+}
+/* ----------------------------------------------------------------- */
+/*                      END FEEDS STYLE                              */
+/* ----------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------- */
+/*                      START CARD STYLE                             */
+/* ----------------------------------------------------------------- */
+.card {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  gap: 2rem;
+  padding: 4rem 8rem;
+  background: #ffffff;
+}
+.card__content {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  box-shadow: 0px 10px 10px #e6e6e6;
+  border-radius: 1rem;
+  margin: 3rem 0;
+}
+.card__image {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  align-items: center;
+  border-top-right-radius: 1rem;
+  border-top-left-radius: 1rem;
+}
+.card__name {
+  text-align: center;
+}
+.card__profession {
+  text-align: center;
+  color: #737373;
+  font-size: small;
+}
+.card__social {
+  display: flex;
+  justify-content: center;
+  gap: 1rem;
+  padding-bottom: 2rem;
+  padding-top: 1rem;
+}
+/* ----------------------------------------------------------------- */
+/*                     CARD   END STYLE                              */
+/* ----------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------- */
+/*                     FOOTER START STYLE                            */
+/* ----------------------------------------------------------------- */
+.footer {
+  padding: 10rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  background: #fff5f0;
+}
+.footer__link {
+  color: #1ba0ab;
+  font-size: small;
+  font-weight: bolder;
+  cursor: pointer;
+  padding: 0.5rem;
+}
+.btn--footer {
+  padding: 10px 20px;
+  border: none;
```

Comment:
You should move this class near other `btn` classes, that's for the code clean up